docker/ts_config/timesketch.conf

```diff
@@ -0,0 +1,12 @@
+DEBUG = False
+SECRET_KEY = u'dockerdockerdocker'
```

Comment:
Ok, now I remembered why I did like this.
Generating a secret during installation is ok, but then it will be statically configured for the built docker container (which then gets pushed to docker hub). That's equally bad I would say.
My original intention was to mount the configuration so you could customise it easily...